ross/shaft_element.py

```diff
@@ -36,7 +36,8 @@ class ShaftElement(Element):
 
     Parameters
     ----------
-
+    L : float, pint.Quantity
+        Shaft length (m).
```

Comment:
One little modification left, instead of `Shaft length` it is the `Element length`.

Reply:
Also done, @marcusfilipesr